kubelet-to-gcm/README.md

```diff
@@ -0,0 +1,85 @@
+# addon-resizer
```

Comment:
Yes. Let me update the README, which will also give context on what this code/daemon is supposed to do.